pyproject.toml

```diff
@@ -43,7 +43,7 @@ exclude = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.10, <3.12"  # no Python 3.12 yet for simpeg-drivers
+python = "^3.10, <3.11"
```

Comment:
**issue:** supposed to work for Python 3.11

Also keep comment to explain why not 3.12